docs/version0.40.md

```diff
@@ -42,6 +43,10 @@ In environments with low physical memory availability, the VM might use too much
 
 For more information, see [`-XX:codecachetotalMaxRAMPercentage`](xxcodecachetotalmaxrampercentage.md).
 
+### Support for persistent shared classes cache added on z/OS
+
+Persistent caches are now supported on z/OS systems. The default cache type on z/OS is still non-persistent. You can use the [`-Xshareclasses: persistent`](xshareclasses.md#persistent) option to switch to persistent shared cache.
```

Comment:
Extra space should be removed.
```suggestion
Persistent caches are now supported on z/OS systems. The default cache type on z/OS is still non-persistent. You can use the [`-Xshareclasses:persistent`](xshareclasses.md#persistent) option to switch to persistent shared cache.
```

Reply:
> Persistent caches are now supported on z/OS systems

I suggest to change "Persistent caches" to "Persistent shared caches".